_python/site_parser.py

```diff
@@ -0,0 +1,95 @@
+import psycopg2 as ps
+import bs4
+import pandas as pd
+import requests
+
+class Webscrapper:
+
+    def __init__(self, url: str) -> None:
+        self.url = url
+
+    def __start_connections(self) -> ps.extensions.connection:
+        try:
+            connection = ps.connect(
+                host = "db",
```

Comment:
Can you ensure that those are provided as env variables in the necessary container? Also, substitute these with `os.getenv`